rclcpp/package.xml

```diff
@@ -4,11 +4,15 @@
   <name>rclcpp</name>
   <version>17.1.0</version>
   <description>The ROS client library in C++.</description>
-  <maintainer email="ivanpauno@ekumenlabs.com">Ivan Paunovic</maintainer>
-  <maintainer email="jacob@openrobotics.org">Jacob Perron</maintainer>
+
   <maintainer email="william@openrobotics.org">William Woodall</maintainer>
+  <maintainer email="ivanpauno@ekumenlabs.com">Ivan Paunovic</maintainer>
+  <maintainer email="michel@ekumenlabs.com">Michel Hidalgo</maintainer>
```

Comment:
The churn here is a little unfortunate; it would be nice if we didn't move Ivan, for instance.  It's not blocking, but maybe it is something we can improve in the code that generates this.

Reply:
I forgot to sort alphabetically, now the churn is much less.